css-positioning/fixed.css

```diff
@@ -0,0 +1,10 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+  position: fixed;
+  left: 35%;
```

Comment:
Can we dynamically center this? In different width of browser, it's not centered